### DISCUSSION TITLE
withResponsiveImages quite buggy when used with multiple conversions

### DISCUSSION THREAD

**Discussion by Admiral-Enigma** [spatie/laravel-medialibrary#3300] [Q&A]
Given this conversion configuration it seems only the xl responsive conversions are persisted on S3.

![image](https://github.com/spatie/laravel-medialibrary/assets/13486531/0b70a63a-2357-4e95-aea9-a0f51df9130e)
Might be related to [https://github.com/spatie/laravel-medialibrary/issues/2378](url)

**Comment by freekmurze** [comment]
I'd accept a PR that fixes this!
